Sources/Soto/Extensions/DynamoDB/DynamoDB+Codable.swift

```diff
@@ -371,10 +377,20 @@ extension DynamoDB {
             let expressionAttributeNames: [String: String]
             if let names = self.expressionAttributeNames, self.updateExpression != nil {
                 expressionAttributeNames = names
+            } else if let additionalAttributeNames {
+                let tmpAttributeNames: [String: String] = .init(item.keys.map { ("#\($0)", $0) }) { first, _ in return first }
+                expressionAttributeNames = tmpAttributeNames.merging(additionalAttributeNames, uniquingKeysWith: { _, new in new })
```

Comment:
So additional attribute names will override what is extracted from the Codable type? Is there a case where this is a valid situation?

Reply:
Not really, but it could be the only way to override the computed value. Do you think is better avoid overriding the Codable type?

Reply:
I was wondering what the intent was, maybe a comment should indicate whether the codable, or additional attribute takes precedence.

Reply:
I fixed the comment about additionalAttributeNames and additionalAttributeValues.